extensions/amp-social-share/0.1/amp-social-share-config.js

```diff
@@ -80,4 +80,11 @@ const BUILTINS = {
       text: 'TITLE - CANONICAL_URL',
     },
   },
+  system: {
+    shareEndpoint: 'share:',
```

Comment:
Could we call this `navigator-share`? I think that would avoid potential confusion with a reasonable future URL scheme.